src/v/serde/parquet/metadata.cc

```diff
@@ -48,6 +48,48 @@ iobuf encode(time_unit t) {
     return std::move(encoder).write_stop();
 }
 
+iobuf encode(const statistics& stats) {
+    constexpr auto null_count_field_id = thrift::field_id(3);
+    constexpr auto max_value_field_id = thrift::field_id(5);
+    constexpr auto min_value_field_id = thrift::field_id(6);
+    constexpr auto is_max_value_exact_field_id = thrift::field_id(7);
+    constexpr auto is_min_value_exact_field_id = thrift::field_id(8);
```

Comment:
is `4` skipped here deliberately ?

Reply:
Yes 4 is `distinct_count`, which most implementations don't write because it can be expensive.